### DISCUSSION TITLE
最近在光映廣場看到了一隻裂界怪物…

### DISCUSSION THREAD

**Discussion by CelineIgnatius** [share121/inter-knot#1782] [其他]
之前在廣場靠海附近看到了一個人的頭很像花，又綠又黑的，回家一查發現可能是只裂界怪物，還好當時後沒去招惹，有繩匠可以看一下是不是怪物有能力變成普通人嗎？

**Comment by yutong903** [comment]
这个委托我接了，记得给五星好评哦

**Reply by yutong903** [reply]
我找到了，那个怪物叫以骸，挺危险的，听安比说，还有的以骸能变成人的模样，一定要小心口牙！

**Comment by ZhuYuan-148** [comment]
这位市民，依骸是不会出空洞的，建议少看点电影。
